app/src/main/java/cm/aptoide/pt/home/bundles/appcoins/FeaturedAppcBundleAdapter.kt

```diff
@@ -0,0 +1,46 @@
+package cm.aptoide.pt.home.bundles.appcoins
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import cm.aptoide.pt.R
+import cm.aptoide.pt.home.bundles.apps.AppInBundleViewHolder
+import cm.aptoide.pt.home.bundles.base.HomeBundle
+import cm.aptoide.pt.home.bundles.base.HomeEvent
+import cm.aptoide.pt.view.app.AppViewHolder
+import cm.aptoide.pt.view.app.Application
+import rx.subjects.PublishSubject
+import java.text.DecimalFormat
+
+class FeaturedAppcBundleAdapter(var apps: List<Application>,
+                                val oneDecimalFormatter: DecimalFormat,
+                                val appClickedEvents: PublishSubject<HomeEvent>) :
+    RecyclerView.Adapter<RecyclerView.ViewHolder>() {
+
+  private var homeBundle: HomeBundle? = null
+  private var bundlePosition: Int = -1
```

Comment:
Is it safe to use this position variable? I know there is not too much to do as the AppsBundleViewHolder receives the position..